install.js

```diff
@@ -18,9 +29,25 @@ async function install() {
     })
     .catch((e) => {
       console.error(`Installation failed: ${e}`);
-      console.error(
-        `Check that you have access to https://github.com/saucelabs/saucectl/releases or format of the SAUCECTL_INSTALL_BINARY environment variable is correct\n\n`,
-      );
+      const binarySource = process.env.SAUCECTL_INSTALL_BINARY;
+      const binarySourceMirror = process.env.SAUCECTL_INSTALL_BINARY_MIRROR;
+      if (binarySource) {
+        console.error(
+          `Please verify that you have access to the SAUCECTL_INSTALL_BINARY environment variable (${sanitizeURL(
```

Comment:
This is kind of awkwardly phrased. It sounds like the problem is the env variable couldn't be read, but the real problem is the url *defined* by the env variable couldn't be accessed.

Reply:
You're also not distinguishing between a network error and any other kind of error. So these error messages might be printed for a totally unrelated error.